### PR TITLE
Add support for grouped digits

### PR DESCRIPTION
**Pull request by ngeri**

### CODE REVIEW

**Comment by ngeri** [comment]
Fixes #13

File: ScrollCounter/NumberScrollCounter.swift [hunk 1]
```diff
@@ -278,6 +284,20 @@ public class NumberScrollCounter: UIView {
         
         let startingX = startingXCoordinate
         let seperatorLocation = digitScrollers.count - decimalPlaces
+
```
Comment (ngeri) [inline ScrollCounter/NumberScrollCounter.swift:287]: I am not satisfied with this solution at all. If you have better idea, please share it with me!

**Comment by stokatyan** [comment]
Hi @ngeri !  I appreciate the contribution to ScrollCounter, and I'm sorry for getting to this so late.

The changes look safe, but can you post a gif of the new feature and a small summary in the description of this PR?

**Comment by ngeri** [comment]
Hey! I have so much work nowdays. I will get back to you with this once I
have some free time again. In the meantime we could close this PR because
it is not complete. I will open a new one!

Thanks!

**Comment by stokatyan** [comment]
Closing this PR by request: https://github.com/stokatyan/ScrollCounter/pull/13#issuecomment-800133284

@ngeri No pressure and no rush, I appreciate you contributing to the project!

**Pull request closed without merging**